en/docs/reference/product-apis/publisher-apis/publisher-v2/publisher-v2.md

```diff
@@ -2,5 +2,5 @@
 template: templates/redoc.html
 ---
 
-<redoc spec-url='{{base_path}}/reference/product-apis/publisher-apis/publisher-v2/publisher-v2.yaml'></redoc>
+<redoc spec-url='https://apim.docs.wso2.com/en/4.0.0-dev/reference/product-apis/publisher-apis/publisher-v2/publisher-v2.yaml'></redoc>
```

Comment:
why is this necessary?

Reply:
Because, we have used `https://apim.docs.wso2.com/en/next` in mkdocs.yml. So I'm trying to figure out whether it is causing the issue.